build/pkgs/python_flint/install-requires.txt

```diff
@@ -0,0 +1 @@
+python-flint
```

Comment:
Do you want to change the file name to `version_requirements.txt` here in preparation for #36999?